### DISCUSSION TITLE
谈谈计算机安全领域的高频词汇 Canary

### DISCUSSION THREAD

**Discussion by giscus** [edony-ink/comments#24] [General]
# 谈谈计算机安全领域的高频词汇 Canary

解决我的一个好奇心：为什么在 Cyber Security 领域中 Canary 是一个高频词汇，它是不是计算机安全领域里面的 “Hello World”？

https://www.edony.ink/cyber-security-canary/

**Comment by edonyzpc** [comment]
今年开始会在计算机安全领域做一下学习和探索
